lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts

```diff
@@ -54,6 +54,9 @@ export class ContentNodeSelectorPanelComponent implements OnInit, OnDestroy {
         hasMoreItems: false
     });
 
+    private _showDropdownSiteList = true;
```

Comment:
I don't think we are using this naming convention of the underscore for private

Reply:
If you expand a bit down there is one _chosenNode:

Reply:
_chosenNode is different has a get and set in that case is needed

Reply:
@pionnegru thanks